docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    Updating
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    RollingRestart
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    Resuming
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  False
+          Type:                    ScalingDown
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  True
+          Type:                    Valid
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  True
+          Type:                    Initialized
+          Last Transition Time:    2022-01-31T19:12:33Z
+          Message:
+          Reason:
+          Status:                  True
+          Type:                    Ready
+        Last Server Node Started:  2022-01-31T19:11:12Z
+        Node Statuses:
+          demo-dc3-default-sts-0:
+            Host ID:  2cceff49-6df2-4045-8e04-6ce262bd6fc4
+          demo-dc3-default-sts-1:
+            Host ID:  018bfd17-8a77-43c6-859b-ab69c1fc8a66
+          demo-dc3-default-sts-2:
+            Host ID:          38438f65-b10a-4b3f-a56f-488536bf4cd3
+        Observed Generation:  1
+        Quiet Period:         2022-01-31T19:12:39Z
+        Super User Upserted:  2022-01-31T19:12:34Z
+        Users Upserted:       2022-01-31T19:12:34Z
+Events:                       <none>
+```
+
+### Extract credentials
+
+On the control plane, use the following commands to extract the username and password.
+
+```bash
+kubectx kind-k8ssandra-0
+
+CASS_USERNAME=$(kubectl get secret demo-superuser -n k8ssandra-operator -o=jsonpath='{.data.username}' | base64 --decode)
+
+echo $CASS_USERNAME
+```
+
+**Output:**
+```bash
+demo-superuser
+```
+
+Now obtain the password secret:
+
+```bash
+CASS_PASSWORD=$(kubectl get secret demo-superuser -n k8ssandra-operator -o=jsonpath='{.data.password}' | base64 --decode)
+
+echo $CASS_PASSWORD
+```
+
+**Output example - your value will be different:**
+```bash
+KT-ROFfbD-O9BzWS3Lxq
+```
+
+{{% alert title="Tip" color="success" %}}
+You'll use the extract credentials for subsequent authentication in deployed containers.
+{{% /alert %}}
+
+### Verify cluster status
+
+On one of the data plane clusters, verify the cluster status. Example:
+
+```bash
+kubectx kind-k8ssandra-1
+
+$ kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
+```
+
+**Output plus nodetool example:**
+
+```bash
```

Comment:
The change above will change this output to something a bit tidier like this:

```
Datacenter: dc1
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
UN  172.18.0.10  327.02 KiB  256          100.0%            9e3e48ee-529e-4b2a-9bf2-39575d32f578  default
UN  172.18.0.11  338.79 KiB  256          100.0%            305616ce-3440-4d37-b9be-32bca624abb7  default
UN  172.18.0.8   304.01 KiB  256          100.0%            0a0864b7-968a-4e07-839f-a5abb2e3d0a4  default
Datacenter: dc2
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
UN  172.18.0.16  355.6 KiB  256          100.0%            f2bd31ef-5ca5-4c28-a9b2-bac28f76af4f  default
UN  172.18.0.15  368.66 KiB  256          100.0%            878e519b-a6f4-4aff-b8ab-c1fb30679847  default
UN  172.18.0.13  344.76 KiB  256          100.0%            3efa6a2f-47d1-49e3-ba93-0a58870e7c0f  default
Datacenter: dc3
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
UN  172.18.0.18  389.07 KiB  256          100.0%            9fc9983e-24a6-415d-9a4d-10d3deff4ca4  default
UN  172.18.0.19  376.66 KiB  256          100.0%            e36d3a62-b521-4a50-b86a-2b392c972195  default
UN  172.18.0.20  401.81 KiB  256          100.0%            396ebb0a-da34-47c9-8f74-0816b5658f86  default
```

Reply:
Got it, thanks. Nice follow-up, appreciated!   Will be in the next commit.